### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by joeedhorton** [UKHomeOffice/design-system#364] [Ideas]
## What
Help users filter data.

## Why
There are internal services that have filtering, however, there are multiple styles used and it would be useful to have guidance and examples.

## Discuss
If you've helped users filter something in your service and tested with users, please add your experiences in the comments below.

## Examples

![filters](https://user-images.githubusercontent.com/19802499/31442496-8810c0e4-ae8e-11e7-825d-b4891fb72ddd.gif)
> Provided by Sam Wakeling. No research available but an interesting approach that could be researched further.

## Related to:
GOV.UK Design System backlog issue:
https://github.com/alphagov/govuk-design-system-backlog/issues/133

**Comment by mat-walker** [comment]
I worked on a filter pattern that I inherited on Atlas which tested ok with users. Whats the best way to collate all these patterns?

**Comment by joeedhorton** [comment]
Hi @mat-walker — If you share them on here, that would be great. Or if they are sensitive designs, you can email them to us at designops@digital.homeoffice.gov.uk and we can start collating examples from there :)

**Comment by karypun** [comment]
Linked this this issue https://github.com/UKHomeOffice/design-system/issues/264
